lib/ts/recipe/session/index.ts

```diff
@@ -67,9 +67,15 @@ export default class SessionWrapper {
         disableAntiCsrf: boolean = false,
         userContext: any = {}
     ) {
-        const claimsAddedByOtherRecipes = Recipe.getInstanceOrThrowError().getClaimsAddedByOtherRecipes();
+        const recipeInstance = Recipe.getInstanceOrThrowError();
+        const claimsAddedByOtherRecipes = recipeInstance.getClaimsAddedByOtherRecipes();
+        const appInfo = recipeInstance.getAppInfo();
+        const issuer = appInfo.apiDomain.getAsStringDangerous() + appInfo.apiBasePath.getAsStringDangerous();
 
-        let finalAccessTokenPayload = accessTokenPayload;
+        let finalAccessTokenPayload = {
+            ...accessTokenPayload,
+            iss: issuer,
+        };
```

Comment:
should this also be added in the `createNewSession` version of the function?

Reply:
It's there:
https://github.com/supertokens/supertokens-node/pull/546/files#diff-2eafc0f3a38c2f8fd37a72bdb09f6d55a8b11d21a3aea625c94845e4a1232162R354